gtest/src/wasm_executor.rs

```diff
@@ -293,6 +332,23 @@ mod meta_tests {
         system
             .0
             .borrow_mut()
-            .call_meta(&program.id, None, void_function_name);
+            .call_meta(&program.id, None, void_function_name)
+            .expect("Meta_state failed");
+    }
+
+    #[test]
+    #[should_panic(
```

Comment:
Isn't it better to use `assert_eq` with the function result instead of `should_panic`?

Reply:
Done with `assert!(matches!(...))`.